cla_backend/apps/checker/views.py

```diff
@@ -44,8 +44,6 @@ class Meta(ArticleCategoryFilter.Meta):
 
 
 class ArticleViewSet(PublicAPIViewSetMixin, BaseArticleViewSet):
-    paginate_by_param = "page_size"
```

Comment:
Why wasn't `pagination_class` added to this viewset?

Reply:
Not sure, I will follow up on this one

Reply:
Checked and this was not doing anything - BaseArticleViewSet already declares the same pagination details